api/views.py

```diff
@@ -128,5 +123,5 @@ def session_list(request):
             status=status.HTTP_404_NOT_FOUND
         )
 
-    serializer = UserSerializer(user)
-    return Response(serializer.data)
+    serializer = UserTripSerializer(user)
```

Comment:
changed to show trip data in resonse

Reply:
nice job implementing this, looks good to me. i think the changes you made to the views are going to break a lot of the tests i just wrote, but such is life 😂